bin/configInitializer.js

```diff
@@ -0,0 +1,131 @@
+"use strict";
+
+var fs = require("fs"),
+    inquirer = require("inquirer"),
+    yaml = require("js-yaml");
+
+/* istanbul ignore next: no need to test inquirer*/
+/**
+ * Ask use a few questions on command prompt
+ * @param {function} processConfigFn function pointer to process user's answers
+ * @param {function} createConfigFileFn function pointer to write result config file
+ * @param {function} callback callback function when file has been written
+ * @returns {void}
+ */
+function prompt(processConfigFn, createConfigFileFn, callback) {
+    inquirer.prompt([
+        {
+            type: "list",
+            name: "indent",
+            message: "What style of indentation do you use?",
+            default: "tabs",
+            choices: [{name: "Tabs", value: "tabs"}, {name: "Spaces", value: 4}]
+        },
+        {
+            type: "list",
+            name: "quotes",
+            message: "What quotes do you use for strings?",
+            default: "double",
+            choices: [{name: "Double", value: "double"}, {name: "Single", value: "single"}]
+        },
+        {
+            type: "list",
+            name: "linebreak",
+            message: "What line endings do you use?",
+            default: "unix",
+            choices: [{name: "Unix", value: "unix"}, {name: "Windows", value: "windows"}]
+        },
+        {
+            type: "confirm",
+            name: "semi",
+            message: "Do you require semicolons?",
+            default: true
+        },
+        {
+            type: "confirm",
+            name: "es6",
+            message: "Are you using ECMAScript 6 features?",
+            default: false
+        },
+        {
+            type: "checkbox",
+            name: "env",
+            message: "Where will your code run?",
+            default: ["browser"],
+            choices: [{name: "Node", value: "node"}, {name: "Browser", value: "browser"}]
+        },
+        {
+            type: "confirm",
+            name: "jsx",
+            message: "Will you use JSX?",
+            default: false
+        },
+        {
+            type: "confirm",
+            name: "react",
+            message: "Will you use React",
+            default: false,
+            when: function (answers) {
+                return answers.jsx;
+            }
+        },
+        {
+            type: "list",
+            name: "format",
+            message: "What format do you want your config file to be in?",
+            default: "JSON",
+            choices: ["JSON", "YAML"]
+        }
+    ], function(answers) {
+        processConfigFn(answers, createConfigFileFn, callback);
+    });
```

Comment:
I think this callback is doing too much. This function should just gather answers and call a single callback. It should be up to the caller to decide what to do.